app/models/history.rb

```diff
@@ -113,7 +118,10 @@ def self.report_reviewed(patient: nil, created_by: 'Sara Alert System', comment:
   end
 
   def self.report_reminder(patient: nil, created_by: 'Sara Alert System', comment: 'User sent a report reminder to the monitoree.')
-    create_history(patient, created_by, HISTORY_TYPES[:report_reminder], comment) unless patient&.preferred_contact_method.nil?
+    return if patient&.preferred_contact_method.nil? && patient&.id == patient&.responder_id
```

Comment:
I'm not sure that we want this logic to be handled/checked in History.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/708/commits/1591d9323747c1f1b05afff99a82e7844ed140ce
Addressed in this commit -- will be moving everything over to this combined PR.